src/components/Sidebar/readme.md

```diff
@@ -48,3 +48,64 @@ class SimpleSidebar extends React.Component {
     </SideBarContainer>
 </div>
 ```
+
+##### Sidebar with dynamic content
+
+```js
+import React from 'react';
+import { Sidebar, SidebarItem } from 'react-rainbow-components';
+import styled from 'styled-components';
+import DynamicContent from './dynamicContent';
+import StyledDynamicContent from './styled/dynamicontent';
+import RenderIf from './../RenderIf';
+
+const SideBarContainer = styled.div.attrs(props => {
+    return props.theme.rainbow.palette;
+})`
+    background: ${props => props.background.main};
+    width: 120px;
+    border-bottom-left-radius: 0.875rem;
+`;
+
+class SimpleSidebar extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedItem: '',
+        };
+        this.handleOnSelect = this.handleOnSelect.bind(this);
+    }
```

Comment:
we should add new examples using functional components with hooks instead of class components